docs/MapColonies/Raster/Guides/raster-export.mdx

```diff
@@ -0,0 +1,247 @@
+                <mc:description>World Wide 2020</mc:description>
+                <mc:footprint>{"type":"Polygon","coordinates":[[[-180,-90],[-180,90],[180,90],[180,-90],[-180,-90]]]}</mc:footprint>
+                <mc:minHorizontalAccuracyCE90>3</mc:minHorizontalAccuracyCE90>
+                <mc:id>3b3164a7-280d-4392-a82f-d60a76e69092</mc:id>
+                <mc:ingestionDate>2022-02-13T13:04:23Z</mc:ingestionDate>
+                <mc:insertDate>2022-02-13T13:04:41Z</mc:insertDate>
+                <mc:layerPolygonParts>{"bbox":[],"type":"FeatureCollection","features":[{"type":"Feature","geometry":{"type":"Polygon","coordinates":[[[],[]]]},"properties":{}}]}</mc:layerPolygonParts>
+                <mc:links scheme="WMS" name="ORTHOPHOTO_MOSAIC_BASE" description="">'<YOUR_MAPPROXY_URL>/service?REQUEST=GetCapabilities'</mc:links>
+                <mc:links scheme="WMS_BASE" name="ORTHOPHOTO_MOSAIC_BASE" description="">'<YOUR_MAPPROXY_URL>/wms'</mc:links>
+                <mc:links scheme="WMTS" name="ORTHOPHOTO_MOSAIC_BASE" description="">'<YOUR_MAPPROXY_URL>/wmts/1.0.0/WMTSCapabilities.xml'</mc:links>
+                <mc:links scheme="WMTS_KVP" name="ORTHOPHOTO_MOSAIC_BASE" description="">'<YOUR_MAPPROXY_URL>/service?REQUEST=GetCapabilities&SERVICE=WMTS'</mc:links>
+                <mc:links scheme="WMTS_BASE" name="ORTHOPHOTO_MOSAIC_BASE" description="">'<YOUR_MAPPROXY_URL>/wmts'</mc:links>
+                <mc:maxResolutionMeter>0.1</mc:maxResolutionMeter>
+                <mc:producerName>IDFMU</mc:producerName>
+                <mc:productBBox>-180,-90,180,90</mc:productBBox>
+                <mc:productId>ORTHOPHOTO_MOSAIC_BASE</mc:productId>
+                <mc:productName>אורתופוטו מתעדכן</mc:productName>
+                <mc:productType>OrthophotoBest</mc:productType>
+                <mc:productVersion>1.0</mc:productVersion>
+                <mc:region>World</mc:region>
+                <mc:maxResolutionDeg>0.000171661376953125</mc:maxResolutionDeg>
+                <mc:sensors>UNDEFINED</mc:sensors>
+                <mc:imagingTimeEndUTC>2020-05-21</mc:imagingTimeEndUTC>
+                <mc:imagingTimeBeginUTC>2020-05-21</mc:imagingTimeBeginUTC>
+                <mc:SRS>4326</mc:SRS>
+                <mc:SRSName>WGS84GEO</mc:SRSName>
+                <mc:transparency>OPAQUE</mc:transparency>
+                <mc:type>RECORD_RASTER</mc:type>
+                <mc:updateDateUTC>2022-02-13T13:03:07Z</mc:updateDateUTC>
+                <ows:BoundingBox crs="urn:x-ogc:def:crs:EPSG:6.11:4326" dimensions="2">
+                    <ows:LowerCorner>-180.0 90.0</ows:LowerCorner>
+                    <ows:UpperCorner>180.0 90.0</ows:UpperCorner>
+                </ows:BoundingBox>
+            </mc:MCRasterRecord>
+        </csw:SearchResults>
+    </csw:GetRecordsResponse>
+  ```
+</details>
+
+## Use Export Service (Step 2)
+Now you need to access export service Export and request export for the desired layer
```

Comment:
PLease change to: "Now you need to make an export request for the desired layer."